charts/yet-another-cloudwatch-exporter/templates/deployment.yaml

```diff
@@ -45,9 +45,13 @@ spec:
           volumeMounts:
             - name: vol-yet-another-cloudwatch-exporter
               mountPath: /config
+        {{- if or (.Values.aws.secret.name) (and .Values.aws.aws_secret_access_key .Values.aws.aws_access_key_id) (and (.Values.extraEnv)) }}
```

Comment:
This might be a bit hard to read... what if we change to always have an `env:` block, even if it ends up being empty? e.g. something like

```
env:
  {{ if set of vars }}
    something
  {{ end }}
  {{ if another set of vars }}
    something else
  {{ end }}
  {{ if extraEnv }}
    extra envs
  {{ end }}
```

Reply:
I also wanted to add proxy settings within the env variables to connect to AWS apis through cooperate environment. So better if we can get this reviewed ASAP. Thanks.

Reply:
Hi, i think it would actually good to introduce an env variable to add proxy for an Enterprise environment. Do we have any fix to introduce proxy in Helm cahrt? or any workaround ?